keyboards/adpenrose/kintsugi/keymaps/franky/keymap.c

```diff
@@ -0,0 +1,173 @@
+/* Copyright 2021 adpenrose
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+#include <stdio.h>
```

Comment:
```suggestion
```

Reply:
I'm using that library so that I can access the sprintf function, which I use in my WPM rendering function. The official documentation states that, while the code that does not require stdio.h saves space, my code is still valid.

Reply:
Sprintf library is massive for AVR.  It adds ~1.5k, even with LTO enabled. 

The `get_u8_str` function was recently added to support stuff like printing the value of WPM to the screen with some char value trickier.  It uses up a LOT less space firmware wise.  which is why.